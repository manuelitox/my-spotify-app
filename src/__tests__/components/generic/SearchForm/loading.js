import React from 'react'
import { shallow } from 'enzyme'

import LoadingMessage from 'components/generic/SearchForm/loading'

describe('Component | LoadingMessage', () => {
  let component 
  beforeEach( () => {
    component = shallow(<LoadingMessage loading />)
  })
  it('should render a component', () => {
    expect(component).toMatchSnapshot()
    expect(component.text()).toEqual('loading')
  })
  it('should render null because isOpen is false', () => {
    component = shallow(<LoadingMessage loading={ false } />)    
    expect(component).toMatchSnapshot()
  })
})