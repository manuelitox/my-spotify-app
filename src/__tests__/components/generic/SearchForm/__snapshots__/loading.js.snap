// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component | LoadingMessage should render a component 1`] = `
<styled.div>
  loading
</styled.div>
`;

exports[`Component | LoadingMessage should render null because isOpen is false 1`] = `""`;
